src/driver-sinowealth.c

```diff
@@ -33,7 +33,7 @@
 #define SINOWEALTH_CONFIG_SIZE_USED_MIN 131
 #define SINOWEALTH_CONFIG_SIZE_USED_MAX 167
 
-#define SINOWEALTH_XY_INDEPENDENT 0x80
+#define SINOWEALTH_XY_INDEPENDENT 0b1000
```

Comment:
commenting on the commit message here (which github doesn't let you): it's "nibbles", not "nimbles".

Reply:
Oh, so that's why spell checker was bugging me... Thank you, fixed!